lib/postman-sandbox.js

```diff
@@ -98,6 +98,7 @@ class PostmanSandbox extends UniversalVM {
             cursor = _.clone(_.get(options, 'cursor', {})), // clone the cursor as it travels through IPC for mutation
             resolvedPackages = _.get(options, 'resolvedPackages'),
             debugMode = _.has(options, 'debug') ? options.debug : this.debug,
+            templateName = _.get(options, 'templateName'),
```

Comment:
I think we should throw an error when `this.connectOptions` has `templates` in it, but `templateName` is missing here.